### PR TITLE
Add script to automatically decrypt secrets

### PR DESCRIPTION
**Pull request by sandr0o**
## Purpose
Our secrets required additional flags to be opened
## Approach
Wrote script to automate building and decryption

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ secrets open [<file path>...] [options]
 
 # To encrypt a file or files.
 secrets seal [<file path>...] [options]
+
+# To encrypt for a Jobbatical.
```
Comment (Param-Harrison) [inline README.md:12]: `To encrypt for Jobbatical`

**Review by Param-Harrison — APPROVED** [review]

**Comment by Rauno56** [comment]
Why remove CicleCI build? Is it built anywhere now @sandr0o?

**Comment by sandr0o** [comment]
> Why remove CicleCI build? Is it built anywhere now @sandr0o?

It is built in google cloud. Also CircleCI was broken, tests were executed but not authenticated

**Comment by Rauno56** [comment]
I see... IMHO

1. You could compile the defaults into "a Jobbatical's edition" and distribute the binary produced in GCP to your devs;
2. Leave the OSS builds somewhere open for others to use.

... that would be easier for you and reusable for everyone else.

**Pull request merged**